lib/tapioca/dsl/pipeline.rb

```diff
@@ -78,6 +78,7 @@ def run(&blk)
             No classes/modules can be matched for RBI generation.
             Please check that the requested classes/modules include processable DSL methods.
           ERROR
+          raise Thor::Error, ""
```

Comment:
```suggestion
          raise Thor::Error, "Exiting"
```
(or something like that)

Reply:
Oh, is the argument displayed to the user?

(not a blocker, I was suggesting it just to ease possible future troubleshooting. If we saw that without any backtrace then it could be pain to track where it came from).